src/components/metadata/MetaSimple.js

```diff
@@ -76,7 +76,7 @@ export class MetaSimple extends Component {
         <TextareaAutosize
           onChange={(e) => this.handleEditableChange(e)}
           className="field value-field"
-          defaultValue={fieldValue}
+          value={fieldValue}
```

Comment:
this needs interpolation to convert `Boolean` to literal.. (detected via Browser Console)
```
value = {`${fieldValue}`}
```